data-sources/prometheus.md

```diff
@@ -1,12 +1,78 @@
 # Prometheus
 
-type: time series
-granularity:
+[Prometheus](https://prometheus.io/) is an open source systems monitoring and alerting toolkit. Prometheus provides time series data identified by metric name and key/value pairs for various services and applications being monitored.
 
-## Long term storage
+**Data Type**: time series
 
-thanos
+**Data Granularity**: seconds, minutes, hours, days
+
+**Users**: SRE (Site Reliability Engineering) teams, DevOps teams, Data Scientists
+
+**Example Prometheus Instance**: http://demo.robustperception.io:9090/graph
+
+![prometheus-timeseries-graph](../assets/images/prometheus-timeseries-graph.png)
+
+Shown above is the Prometheus UI which allows you to search and aggregate different time series metrics and view them as a graph.
+
+## Purpose and Use of Prometheus
+
+Prometheus monitoring has become the industry standard and is implemented by software companies worldwide. It supports multidimensional data collection and data queuing, and is reliable, allowing customers to quickly diagnose problems.
+
+**Operations Perspective** - From an operations standpoint, Prometheus is useful in tracking the performance of various services/applications and enables SRE teams to effectively diagnose, troubleshoot and handle incidents for any degradation of services.
+
+**Data Science Perspective** - From a Data Science standpoint, Prometheus provides a large collection of time series data which can be made available to Data Scientists for exploring and applying ML methods to solve unique problems such as time series forecasting to predict when a possible system failure may occur in the future or anomaly detection i.e identifying any erratic or erroneous system behavior.
```

Comment:
From a Data Science standpoint, Prometheus provides a large collection of time series data for exploring and applying ML methods to solve unique problems such as forecasting to predict a future system failure or anomaly detection to identify erratic or erroneous system behavior.